### PR TITLE
babbage spec - two todo items

### PR DESCRIPTION
**Pull request by JaredCorduan**
This PR complete two items from the Babbage spec TODO. Each is in a separate commit (and is probably easier to review commit by commit).

* Single VRF check
* Address Shelley errata 17.2, namely do not prefilter the reward update by the registered stake credentials.

### CODE REVIEW

File: eras/babbage/formal-spec/rewards.tex [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The reward calculation no longer filters out the unregistered stake
+credentials when creating a reward update. As in the Shelley era, though,
+they are still filtered on the epoch boundary when the reward update is applied.
+This addresses errata 17.2 in the Shelley ledger specification \cite{shelley_spec}[17.2].
+The change consists of removing the line
+$$\var{addrs_{rew}}\restrictdom{\var{potentialRewards}}$$
+from the $\fun{rewardOnePool}$ function.
+
+\begin{figure}[htb]
```
Comment (JaredCorduan) [inline eras/babbage/formal-spec/rewards.tex:13]: It seems a real shame to have to add this big table for such a simple edit (removing one short line and renaming one variable). Perhaps we can do better? Just describe the diff? :thinking:

Comment (WhatisRT) [reply eras/babbage/formal-spec/rewards.tex:13]: Yeah, I'd put three dots for everything in the where block except the last line, and maybe highlight that one

Comment (JaredCorduan) [reply eras/babbage/formal-spec/rewards.tex:13]: I like that much better, thanks!

**Review by WhatisRT — APPROVED** [review]
I'd add these to the list of changes in the introduction, otherwise LGTM!

**Comment by JaredCorduan** [comment]
> I'd add these to the list of changes in the introduction, otherwise LGTM!

great idea, thanks!

**Review by WhatisRT — APPROVED** [review]

File: eras/babbage/formal-spec/introduction.tex [hunk 2]
```diff
@@ -23,5 +21,8 @@ \section{Introduction}
 \item Add inline datums in the UTxO
 \item Add reference scripts
 \item Add transaction fields for specifying and returning collateral
-\item Remove the protocol parameter $\var{d}$ and the overlay schedule
+\item Remove the protocol parameters $\var{d}$ and $\var{extraEntropy}$
```
Comment (JaredCorduan) [inline eras/babbage/formal-spec/introduction.tex:24]: @WhatisRT just letting you know I snuck this in as well (mentioning removing the extra entropy).

**Pull request merged**